### DISCUSSION TITLE
Mobox for Apple/Consoles/Linux ecosystem...!?

### DISCUSSION THREAD

**Discussion by 6eppicc9** [olegos2/mobox#322] [General]
Hello! 
I want to know if will be created/ported for IOS/MACOS (or maybe LINUX & CONSOLE) too, because I want to move on ANDROID only for this project/option to play WINDOWS GAMES on MOBILE... 
In any case, we BIG THANK the creators enormously for fulfilling this dream of many I think I could say, I wish them good health and great success in the future... ALL THE BEST TO EVERYONE!

**Comment by qwdnj** [comment]
I feel too good

**Reply by Soranim4296** [reply]
Great! Thank you
